dotnet/samples/Demos/VectorStoreRAG/Options/OpenAIConfig.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.ComponentModel.DataAnnotations;
+
+namespace VectorStoreRAG.Options;
+
+/// <summary>
+/// OpenAI service settings.
+/// </summary>
+internal sealed class OpenAIConfig
+{
+    public const string ConfigSectionName = "OpenAI";
+
+    [Required]
+    public string ModelId { get; set; } = string.Empty;
+
+    [Required]
+    public string ApiKey { get; set; } = string.Empty;
```

Comment:
There should also be a property for the OrgId

Reply:
Thanks, fixed.